src/@awesome-cordova-plugins/plugins/purchases/index.ts

```diff
@@ -168,36 +291,28 @@ export enum INTRO_ELIGIBILITY_STATUS {
  * | 📮  | Great support - [Help Center](https://revenuecat.zendesk.com)                                                                                           |
  * | 🤩  | Awesome [new features](https://trello.com/b/RZRnWRbI/revenuecat-product-roadmap)                                                                             |
  *
- * ## Getting Started
- *
- * For more detailed information, you can view our complete documentation at [docs.revenuecat.com](https://docs.revenuecat.com/docs).
- * @usage
- * #### 1. Get a RevenueCat API key
+ * ## Capacitor
  *
- * Log in to the [RevenueCat dashboard](https://app.revenuecat.com) and obtain a free API key for your application.
+ * The Cordova plugin is not guaranteed to be up to date and we are moving our efforts to the Capacitor plugin.
+ * Please consider using our [Capacitor plugin](https://docs.revenuecat.com/docs/capacitor) instead of the Cordova plugin and this wrapper.
  *
- * #### 2. Initialize the SDK
+ * ## Getting Started
  *
- * You should only configure _Purchases_ once (usually on app launch) as soon as your app has a unique user id for your user. This can be when a user logs in if you have accounts or on launch if you can generate a random user identifier.
+ * Please follow the [Quickstart Guide](https://docs.revenuecat.com/docs/) for more information on how to use the SDK * @usage
```

Comment:
what does ` * @usage` do here?

Reply:
Oh good catch this used to be in line 174, it got moved here by mistake. I think it determines the end of what appears in their official docs 

https://danielsogl.gitbook.io/awesome-cordova-plugins/purchases